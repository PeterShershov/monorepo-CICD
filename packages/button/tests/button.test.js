import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import Button from "../src/button";

describe("Button component", () => {
  test("Button changes style onClick", () => {
    const cmp = shallow(<Button />);
    expect(cmp.hasClass("button_active")).toEqual(false);
    cmp.simulate("click");
    expect(cmp.hasClass("button_active")).toEqual(true);
  });
});
