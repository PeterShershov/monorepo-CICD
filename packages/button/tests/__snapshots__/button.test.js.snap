// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Button renders children 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  TEST
  TEST
</button>
`;
